apps/common/controllers/address-lookup-loop.js

```diff
@@ -47,12 +27,12 @@ module.exports = class AddressLookupLoopController extends BaseController {
     if (req.params.action === 'edit') {
       const steps = req.sessionModel.get('steps');
       _.remove(steps, step => {
```

Comment:
I agree as a general programming principle, but I'd leave it for another day in the interests of getting this in. It works as it is